codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ServerOperationRegistryGenerator.kt

```diff
@@ -164,6 +164,7 @@ ${operationImplementationStubs(operations)}
     }
 
     private fun renderOperationRegistryStruct(writer: RustWriter) {
+        writer.rust("""##[deprecated(since = "0.52.0", note = "OperationRegistry is part of the deprecated service builder API. Use `$serviceName::builder` instead.")]""")
```

Comment:
```suggestion
        writer.rust("""##[deprecated(since = "0.52.0", note = "`OperationRegistry` is part of the deprecated service builder API. Use `$serviceName::builder` instead.")]""")
```